new.php

```diff
@@ -113,6 +115,20 @@
 
 $wikiName = "Patch Demo ($patchesAppliedText)";
 
+$mainPage = "This wiki was automatically generated on [$server$serverPath Patch Demo] at ~~~~~ and applies the following patches:\n";
```

Comment:
`applies` should be `applied` for consistency with `was automatically generated`
But, the wiki wasn't "automatically" generated - a user had to choose to created it, so perhaps `was generated`?

Reply:
Removed "automatically". I'm using "applies" in the continuous tense in that the server makes use of (and continues to make use of) the patches.